CHANGELOG.md

```diff
@@ -1,36 +1,90 @@
 ## 1.8.4 (Upcoming)
+
 ### NOTES:
-* The Oracle builder and post-processor are no longer vendored with Packer core,
-     users of the Oracle plugin should use `packer init` to install the latest
-     version of the plugin. See the [Oracle Plugin
+
+* Packer user documentation has been moved to the new HashiCorp developer
+     documentation portal. The main Packer site (https://www.packer.io) will
+     continue to be the home for Packer but all requests for general
+     documentation and binary downloads will be redirected to
+     https://developer.hashicorp.com/packer. Users are encouraged to visit the
+     developer documentation portal for access to all Packer related
+     documentation; including integrations with HCP Packer.
+
+* The Oracle builder and post-processor are no longer vendored with Packer
+     core. Users of the Oracle plugin should use `packer init` to install the
+     latest version of the plugin. See the [Oracle Plugin
      Documentation](https://github.com/hashicorp/packer-plugin-oracle) for more
      information. [GH-11983](https://github.com/hashicorp/packer/pull/11983)
 
+* HCP Packer environment variables: The behaviour of some HCP Packer-specific
+     environment variables has changed slightly. Refer to [HCP Packer](https://developer.hashicorp.com/packer/docs/hcp)
+     in the Packer documentation for a full list of HCP Packer environment variables.
+    - For JSON templates, the `HCP_PACKER_REGISTRY` environment variable was
+     previously required to enable the HCP Packer integration. In this release,
+     the environment variable is now optional, and can be used for disabling the
+     publishing of metadata for any HCP Packer enabled configuration template.
+    - For HCL2 templates, the `HCP_PACKER_REGISTRY` environment variable can be used
+     to disable publishing to a HCP Packer registry even if the template defines a
+     `hcp_packer_registry` block. This can be useful for testing that a template
+     works as intended prior to pushing metadata to HCP Packer.
+    - For JSON and HCL2 templates, setting the `HCP_PACKER_BUCKET_NAME` environment
```

Comment:
This feels out-of-place in a changelog as this is already the existing behaviour, we can however highlight that `HCP_PACKER_BUCKET_NAME` alone is enough for enabling HCP?

Something like:
```
The `HCP_PACKER_BUCKET_NAME` environment variable alone is now the only requirement to push metadata from a build to HCP, in both JSON templates and HCP templates without a `hcp_packer_registry` block.
```

Reply:
Good call. I'll update to reflect your suggested change. Ty